contracts/common/GetActionSummary.sol

```diff
@@ -170,10 +166,8 @@ contract GetActionSummary is ExtractCallData, GetActionDomainSkillId {
         }
       } else {
         // If no expenditure signatures have been seen, we record the latest signature
-        // unless we're already flagged as a NO_ACTION or OLD_MOVE_FUNDS
         // Also, we aggregate the permissions as we go
-
-        if (!isExpenditureSig(totalSummary.sig) && !isSpecialFunction(totalSummary.sig)) {
+        if (!isExpenditureSig(totalSummary.sig)) {
```

Comment:
Is this signature check necessary? Haven't we already established that this is false?

Reply:
I don't believe so? We've established that `actionSummary.sig` is not an `ExpenditureSig`, but not `totalSummary.sig`.